Scripts/HTMLtoMD/HTMLtoMD.py

```diff
@@ -0,0 +1,35 @@
+import demistomock as demisto
+from CommonServerPython import *
+from markdownify import markdownify as md
+
+
+def html_to_md_command(args):
+    html = str(args.get('html'))
```

Comment:
can you str() a None? please handle

Reply:
The argument is listed as mandatory so I set a default empty string if none is provided.